src/globals/theme/_index.scss

```diff
@@ -10,10 +10,12 @@
 @import '../deprecate/index';
 @import '../feature-flags/index';
 
-@import 'functions';
 @import 'variables';
 
-// TODO: `2.x` - To be removed.
+// Theme blacklist.
+/// @type List<String>
+/// @access private
+$security--theme__blacklist: ();
```

Comment:
Could you please rename this? I suggest a "blocklist", "exclusionary list", "list of excluded variables", etc.